Tasks/KubernetesManifestV0/src/utils/DeploymentHelper.ts

```diff
@@ -137,3 +141,59 @@ function updateImagePullSecretsInManifestFiles(filePaths: string[], imagePullSec
 function isCanaryDeploymentStrategy(deploymentStrategy: string): boolean {
     return deploymentStrategy != null && deploymentStrategy.toUpperCase() == canaryDeploymentHelper.CANARY_DEPLOYMENT_STRATEGY.toUpperCase();
 }
+
+function checkPodStatus(kubectl: Kubectl, podName: string) {
+    const startTime = new Date();
+    const timeOut = 5 * 60 * 1000; // Timeout 5 min
+    let currentTime = new Date();
+    let podStatus;
+    while (currentTime.getTime() - startTime.getTime() < timeOut) {
+        tl.debug('Polling for pod status');
+        podStatus = getPodStatus(kubectl, podName);
+        if (podStatus.status && podStatus.phase) {
```

Comment:
getPodStatus returns = Podresult.stdOut.status. Again here what will be the value of podStatus.status ???

Reply:
Good catch. I missed removing it.